docs/index.md

```diff
@@ -99,7 +99,7 @@ fundamentals/concepts
 fundamentals/document/index
 fundamentals/executor/index
 fundamentals/flow/index
-fundamentals/practice-your-learning
+fundamentals/tutorials/index
```

Comment:
tutorials should not be part of fundamentals, as it may contain advanced topics in the future.

Please move it to a new top-level section.

Reply:
maybe you can merge this branch: docs-chatbot-tutorial to yours and add the executor tutorial to the same section

Reply:
Ignore the comment above, I'll merge yours  so that both tutorials fall in the same section